pkg/scale/encode_test.go

```diff
@@ -944,29 +944,6 @@ var (
 		},
 	}
 
-	bitVecTests = tests{
```

Comment:
let's move these tests as well to `types/bitvec_test.go`?

Reply:
These are essentially the same tests as found in `bitvec_test.go`.